index.html

```diff
@@ -39,7 +39,7 @@ <h3 class="text-muted">Chromebrew</h3>
       </div>
 
       <div class="row marketing">
-        <div class="col-lg-6">
+        <div class="col-6">
```

Comment:
I think those lines are under previous line's `div class="row marketing"`, so it won't span the entire page width.  Please try that one to see what it does.  Thanks.

We can say something like `div class="col"` if we don't want to remove these lines completely.  I just thought `"col-6"` is strange.  If it is the common way to mark this situation, I accept that one too.  Please let me know.